logdevice/ops/ldquery/tables/Directories.h

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc. and its affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+#pragma once
+
+#include <map>
+#include <vector>
+
+#include "../Context.h"
+#include "../Table.h"
+
+namespace facebook {
+  namespace logdevice {
+    namespace ldquery {
+      namespace tables {
+
+class Directories : public Table {
+ public:
+  explicit Directories(std::shared_ptr<Context> ctx) : Table(ctx) {}
+  static std::string getName() {
+    return "dirs";
```

Comment:
Same here, `log_dirs` might be a bit more descriptive.

Reply:
ok